### PR TITLE
[LRN] Added WOFF and WOFF2 fonts built from OTF

### PR DESCRIPTION
**Pull request by karoltarasiuk**

### CODE REVIEW

**Comment by laughinghan** [comment]
Sweet! What tool did you use to build the WOFF{,2} files? Also, they're used by non-IE too right, let's double-check those are fine?

**Comment by karoltarasiuk** [comment]
I've used this one: https://font-converter.net/en

I've uploaded OTF and generated WOFF and WOFF2 only. I checked our demo page on Edge, Chrome, Firefox and Safari and looks OK.

**Comment by laughinghan** [comment]
Let's add the test case from #530 to our visual tests: https://github.com/Learnosity/mathquill/pull/49

Mind posting screenshots? I'll take care of OS X (10.10.5):

Chrome 48:
image

Firefox 45:
image

Safari 9.0.3:
image

The scaled-down 5 doesn't look great, but overall there's no glaring problems like there were in IE.

**Comment by karoltarasiuk** [comment]
Edge on Win10:
screen shot 2016-04-01 at 9 51 57 am
Chrome on Win10:
screen shot 2016-04-01 at 9 57 39 am
FF on Win10:
screen shot 2016-04-01 at 9 58 21 am
Safari 5 on Win10 (you can't download newer for Windows from anywhere):
screen shot 2016-04-01 at 10 01 22 am
IE11 on Win8 (error thrown in the console: math field 337px wide instead of 102px):
screen shot 2016-04-01 at 10 07 39 am
Same on IE10 on Win7, and IE9 on Win7
IE8 on Win7:
screen shot 2016-04-01 at 10 09 57 am

Red pages in IEs above are not connected to the font change though, so I don't think it's a blocker for this PR.

**Comment by laughinghan** [comment]
Hahaha IE8 and Edge work but not IE9-11? It looks like the stylesheet didn't load, probably #324

File: test/visual.html [hunk 1]
```diff
@@ -197,6 +197,20 @@
...
 <tr><td><span class="mathquill-static-math">\sum _{n=0}^3\cos x</span><td><span>\sum _{n=0}^3\cos x</span>^M
+<tr><td colspan=2><span id="sixes"></span>
+<script>
+$(function() {
+    var i,
+        sixes = '',
+        start = 10,
+        end = 40;
+    for (i = start; i <= end; i++) {
+        sixes += '<span style="font-size: ' + i + 'px" class="mathquill-math-field">6<\/span>';
```
Comment (laughinghan) [inline test/visual.html]: @karoltarasiuk: just noticed the backslash in `<\/span>`. Fun fact: this was specified to be necessary in HTML4 via SGML, but was never actually necessary in any browser and is unnecessary in HTML5/the HTML Living Standard (summary: http://stackoverflow.com/a/236106/362030 , long version: http://mths.be/etago )

**Pull request merged**